.eslintrc

```diff
@@ -1 +1,54 @@
-{ "extends": "scality" }
+{
+    "extends": "scality",
+    "parserOptions": {
```

Comment:
can't we move these the scality/guidelines package, to avoid copying/duplicating and maintaining different styles?

Reply:
Actually I think we can just remove them and adapt the code. I'll give it a quick try.

Reply:
Done, like that we don't need to change scality guidelines by disabling good practice rules...